pipelines/katello_devel/03-tests.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts:
+  - pipeline-katello-devel-centos7
+  become: yes
+  vars_files:
+  - ../vars/repos_staging.yml
+  roles:
+  - ruby_scl
+  - nodejs_scl
+  - postgresql_scl
```

Comment:
In the `02-install.yaml` step, the SCL packages are already being handled by the inclusion of `katello_devel` role which additionally ensures things are handled properly on EL8 (https://github.com/theforeman/forklift/blob/master/roles/katello_devel/meta/main.yml#L2). If that works, I'd drop these call outs here and let that step handle things. Otherwise, I would go ahead and additionally scope these similar to `katello_devel` to only include these roles on El7. That way, given devel installs can be on El8 and will be there primarily in a few months this is already ready for it.